docs/source/backends/hdf5.rst

```diff
@@ -33,10 +33,11 @@ Environment variable                     Default      Description
 ``OPENPMD_HDF5_PAGED_ALLOCATION_SIZE``   ``33554432`` Size of the page, in bytes, if HDF5 paged allocation optimization is enabled.
 ``OPENPMD_HDF5_DEFER_METADATA``          ``ON``       Tuning parameter for parallel I/O in HDF5 to enable deferred HDF5 metadata operations.
 ``OPENPMD_HDF5_DEFER_METADATA_SIZE``     ``ON``       Size of the buffer, in bytes, if HDF5 deferred metadata optimization is enabled.
-``H5_COLL_API_SANITY_CHECK``             unset        Debug: Set to ``1`` to perform an ``MPI_Barrier`` inside each meta-data operation.
 ``HDF5_USE_FILE_LOCKING``                ``TRUE``     Work-around: Set to ``FALSE`` in case you are on an HPC or network file system that hang in open for reads.
+``HDF5_DO_MPI_FILE_SYNC``                unset        Work-around: Set to ``1`` to overcome MPI-I/O synchronization issues on some filesystems, e.g., NFS & UnifyFS.
```

Comment:
@jeanbez I don´t fully understand the value here.

In
https://github.com/openPMD/openPMD-api/issues/1423#issuecomment-1515070454
you seem to suggest one should turn `HDF5_DO_MPI_FILE_SYNC=FALSE` for NFS - isn´t that the default (to not spill syncs everywhere)?

Reply:
Oh, I checked https://github.com/HDFGroup/hdf5/pull/1801 and it might default to `1` depending on the driver...? :-o

Reply:
Yep, that seems to be correct.

Reply:
Ok, need to rephrase the guidance a bit then.